docs/pages/machine-id/workload-identity/aws-roles-anywhere.mdx

```diff
@@ -0,0 +1,277 @@
+---
+title: Configuring Workload ID and AWS Roles Anywhere
+description: Configuring AWS to accept Workload ID certificates as authentication using AWS Roles Anywhere
+---
+
+Teleport's Workload Identity issues flexible short-lived identities in X.509
+certificates. AWS Roles Anywhere allows you to use these certificates to
+authenticate to AWS services.
+
+This can be useful in cases where a machine needs to securely authenticate with
+AWS services without the use of a long-lived credential. This is because the
+machine can authenticate with Teleport without using any shared secrets by
+using one of our delegated join methods.
+
+This implementation differs from Teleport's Application Access for AWS feature
+in a few ways:
+
+- Requests to AWS are not proxied through Teleport, meaning reduced latency but
+  also less visibility, as these requests will not be recorded in Teleport's
+  audit log.
+- Workload ID works with any AWS client, including the commandline tool but also
+  their SDKs
+- Teleport Application Access for AWS does not work with Machine ID and
+  therefore cannot be used when a machine needs to authenticate with AWS.
+
+Whilst this guide is primarily aimed at allowing a machine to access AWS,
+the `tsh svid issue` command can be used in place of Machine ID to allow a human
+user to authenticate with using AWS Roles Anywhere.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- (!docs/pages/includes/tctl.mdx!)
+- `tbot` must already be installed and configured on the host where the
+workloads which need to access Teleport Workload Identity will run. For more
+information, see the [deployment guides](../deployment.mdx).
+
+## Step 1/5. Decide on a SPIFFE ID structure
+
+Within Teleport Workload Identity, all identities are represented using a
+SPIFFE ID. This is a URI that uniquely identifies the entity that the identity
+represents. The scheme is always `spiffe://`, and the host will be the name of
+your Teleport cluster. The structure of the path of this URI is up to you.
+
+For the purposes of this guide, we will be granting access to AWS to the
+`spiffe://example.teleport.sh/svc/example-service` SPIFFE ID.
+
+If you have already deployed Teleport Workload Identity, then you will already
+have a SPIFFE ID structure in place. If you have not, then you will need to
+decide on a structure for your SPIFFE IDs.
+
+If you are only using Teleport Workload ID with AWS Roles Anywhere, you way
+structure your SPIFFE IDs so that they explicitly specify the AWS role they are
+allowed to assume. However, it often makes more sense to name the workload or
+person that will use the SPIFFE ID. See the
+[best practices guide](./best-practices.mdx) for further advice.
```

Comment:
Since a user needs to leave the guide and do some planning, I would recommend making Step 1 part of the Prerequisites, i.e., we'd require reading the best practices guide and determining a SPIFFE ID structure before following this guide.

Reply:
67680590c3